roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=missing-docstring
```

Comment:
> PyLint rules MUST NOT be disabled on a whole file.
> 
> And I suppose I'd rather have the docstrings, or make the methods internal, than have this everywhere.

I started doing this in checks because every check has a `run` method, and it doesn't make sense to 1) make it "internal" `_run`; nor 2) add a docstring for it in every subclass of `OpenShiftCheck`.

I don't really like dirtying the file with pylint comments. Once I proposed we disable `missing-docstring` globally, but it did not get the discussion it required to go forward.

Thoughts?

> Also from the same doc:
> 
> \# vim: expandtab:tabstop=4:shiftwidth=4

This one smells like something we may want to remove from the rules. I am not sure, but I think in some PR I got a sign from @detiber that this is not something we insist in.

In a project where contributors are free to use whatever editor they want, and we have linting tools that verify the proper formatting of files, I'd say that `vim`-specific line is not needed.